src/main/java/use_case_interactors/EmployeeFacade.java

```diff
@@ -1,11 +1,12 @@
 package use_case_interactors;
 
+
 public class EmployeeFacade implements EmployeeFacadeInterface{
     String userName;
 
     public void addItem(String serialNo, int increasedQuantity)
```

Comment:
Hey Nav,

The overall code is good and looks like it should be able to easily incorporate the implementation of the other features. Here are a few points that you may want to consider as we begin to connect all of our features: 

1. To add to Tony's point about the return type of add item, I think that it would be likely that the add item method would never be unsuccessful (as we remove items in such a way that the amount can never be negative and if the serial number is not already in the inventory, we could just add it as a new value or assume that this case is not possible - for instance, a store would not be able to sell an item not explicitly approved by the head office, so it would not be added) so returning a boolean may not give any additional information. Although, you could consider having the return type be able to give some important information to the user (like how many items are now in stock). 
2. When I finish with the search and sort implementation, I will send you a message so you can fully implement them for the interface, although we may have to discuss what it may look like to "sort" the items.

Reply:
- Yeah, it's a good idea to probably return something that prompts the appearance of a message that says something like add or return successful and then displays the serial no along with the updated quantity. 
- I think for sort we were just gonna do filter I suppose?